Project.toml

```diff
@@ -6,8 +6,10 @@ version = "0.1.0"
 [deps]
 CLIMAParameters = "6eacf6c3-8458-43b9-ae03-caf5306d3d53"
 ClimaCore = "d414da3d-4745-48bb-8d80-42e94e092884"
+DiffEqCallbacks = "459566f4-90b8-5000-8ac3-15dfb0a30def"
```

Comment:
I believe these are just in the test environment for now (the ClimaLSM package is in some sense agnostic to timestepper). we can discuss if this makes sense